// CS1015: Only `object', `string', and types that derive from `System.Exception' can be passed into a catch block
// Line: 9

class Test
{
    public static void Main ()
    {
    	try {}
    	catch (int[]) {}
    }
}

